packages/application-components/src/components/tab-header/tab-header.tsx

```diff
@@ -84,20 +80,16 @@ export const TabHeader = (props: TTabHeaderProps) => {
 
   return (
     <Link
+      role="tab"
+      aria-selected={isActive}
       to={props.to}
       css={getLinkStyles(isActive, isDisabled)}
       {...getDisabledLinkAtributes(isDisabled)}
       {...dataAttributeProps}
     >
-      <div
-        role="tab"
-        aria-selected={isActive}
-        {...getDisabledTabHeaderAriaAttributes(isDisabled)}
-      >
```

Comment:
Can't remember why we implemented it like this 🤔

Reply:
I suppose it's a leftover from [this refactor](https://github.com/commercetools/merchant-center-application-kit/pull/2503#discussion_r826076176)